src/Resources/config/routing/search.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<routes xmlns="http://symfony.com/schema/routing"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+
+    <route id="elasticsearch.api.search" path="/search.json" methods="GET">
```

Comment:
Maybe also define a route with the new naming convention ? emsco_ajax_query_search